client/src/utils/helpers.js

```diff
@@ -17,12 +17,12 @@ var _ = {
   eachAsync: function(obj, fn, cb){
     var c = obj.length;
     
-    var next = function(){
+    var next = function(err){
```

Comment:
True. This is really just a little custom polyfill for some little async functions that I use - mostly to keep the client as much "dependency free" and small as possible. The behaviour I changed here is that for a group of parallel async tasks, I want the cycle to exit not only when all the tasks are done - but **also** when any of the tasks errors via "calling back" an error.